### DISCUSSION TITLE
Q: SITE_NAME is a reserved environment variables

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2605] [Q&A] [answered]
### Describe the bug | 问题描述

如图：
![image](https://github.com/walinejs/waline/assets/90504717/831e8e22-048f-4373-9cce-0b8e00991d90)

### Website URL | 问题网站

https://preeminent-narwhal-b68963.netlify.app/.netlify/functions/comment

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

As shown in the picture:
![Screenshot 2024-07-10 093531](https://github.com/walinejs/waline/assets/90504717/569ef549-328a-44fe-a4c2-bc5447e2f313)

### Website URL | Problem website

https://preeminent-narwhal-b68963.netlify.app/.netlify/functions/comment

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
You can modify your netlify project's name as SITE_NAME directly. https://github.com/orgs/walinejs/discussions/2098
